lib/index.js

```diff
@@ -173,12 +198,20 @@ export default class HyperList {
     }
   }
 
+  _calculatePositions (startFrom) {
+    const itemHeights = this._itemHeights
+    const itemPos = this._itemPositions
+
+    itemHeights.slice(startFrom).forEach((itemHeight, i) => {
```

Comment:
Can you just do a `for (let i = startFrom; i < total; i++)`? `slice` + `forEach` is way slower.